<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-06-14T23:30:09 IST" name="Suite" finished-at="2025-06-14T23:30:30 IST" duration-ms="20961">
    <groups>
    </groups>
    <test started-at="2025-06-14T23:30:09 IST" name="HomePageTestCase" finished-at="2025-06-14T23:30:22 IST" duration-ms="12745">
      <class name="com.CRM.TestCase.HomePageTestCase">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.CRM.TestCase.HomePageTestCase@292b08d6]" started-at="2025-06-14T23:30:09 IST" name="setup" finished-at="2025-06-14T23:30:17 IST" duration-ms="8403" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyUrl()[pri:1, instance:com.CRM.TestCase.HomePageTestCase@292b08d6]" started-at="2025-06-14T23:30:17 IST" name="verifyUrl" finished-at="2025-06-14T23:30:20 IST" duration-ms="2250" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUrl -->
        <test-method signature="verifyTitle()[pri:2, instance:com.CRM.TestCase.HomePageTestCase@292b08d6]" started-at="2025-06-14T23:30:20 IST" name="verifyTitle" finished-at="2025-06-14T23:30:20 IST" duration-ms="23" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTitle -->
        <test-method signature="validateSignInLink()[pri:3, instance:com.CRM.TestCase.HomePageTestCase@292b08d6]" started-at="2025-06-14T23:30:20 IST" name="validateSignInLink" finished-at="2025-06-14T23:30:22 IST" duration-ms="1957" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateSignInLink -->
      </class> <!-- com.CRM.TestCase.HomePageTestCase -->
    </test> <!-- HomePageTestCase -->
    <test started-at="2025-06-14T23:30:22 IST" name="LoginPageTestCase" finished-at="2025-06-14T23:30:30 IST" duration-ms="8046">
      <class name="com.CRM.TestCase.LoginPageTestCase">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.CRM.TestCase.LoginPageTestCase@49438269]" started-at="2025-06-14T23:30:22 IST" name="setup" finished-at="2025-06-14T23:30:28 IST" duration-ms="5949" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="pageSetup()[pri:0, instance:com.CRM.TestCase.LoginPageTestCase@49438269]" started-at="2025-06-14T23:30:28 IST" name="pageSetup" finished-at="2025-06-14T23:30:29 IST" duration-ms="1596" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pageSetup -->
        <test-method signature="validateLogin()[pri:0, instance:com.CRM.TestCase.LoginPageTestCase@49438269]" started-at="2025-06-14T23:30:29 IST" name="validateLogin" finished-at="2025-06-14T23:30:30 IST" duration-ms="483" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Test Fail : Login Fail expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Test Fail : Login Fail expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertTrue(Assert.java:56)
at com.CRM.TestCase.LoginPageTestCase.validateLogin(LoginPageTestCase.java:21)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateLogin -->
      </class> <!-- com.CRM.TestCase.LoginPageTestCase -->
    </test> <!-- LoginPageTestCase -->
  </suite> <!-- Suite -->
</testng-results>
